bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java

```diff
@@ -1236,6 +1237,24 @@ public T setReplicationRateByBytes(int rate) {
         return getThis();
     }
 
+    /**
+     * get the max tasks can be acquired per second of re-replication.
+     * @return max tasks can be acquired per second of re-replication.
+     */
+    public double getReplicationAcquireTaskPerSecond() {
+        return getDouble(REPLICATION_ACQUIRE_TASK_PER_SECOND, 1.0);
```

Comment:
Let's set default to 0 (no throttling) to maintain existing behavior by default

Reply:
Good idea, i have changed the implementation, thanks.